testsuite/integration/smoke/src/test/java/org/jboss/as/test/smoke/jms/JMSBridgeTest.java

```diff
@@ -125,6 +136,7 @@ public void sendAndReceiveMessage() throws Exception {
             assertTrue(receivedMessage instanceof TextMessage);
             assertEquals(text, ((TextMessage) receivedMessage).getText());
             assertNotNull("did not get header set by the JMS bridge", receivedMessage.getStringProperty(ActiveMQJMSConstants.AMQ_MESSAGING_BRIDGE_MESSAGE_ID_LIST));
+            assertEquals("bridge metrics are not correct", 5000L,readMetrics());
```

Comment:
Can you please edit the message slightly so it would be more explaining? It is obvious, that when assert fails, something is not correct/expected...